src/renderers/webgl/WebGLTextures.js

```diff
@@ -177,6 +177,34 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 	}
 
+	function getMipLevels( texture, image, supportsMips ) {
```

Comment:
Looks like we don't need the `level` variable?

```js
function getMipLevels( texture, image, supportsMips ) {

	if ( textureNeedsGenerateMipmaps( texture, supportsMips ) === true ) {

		// generated mipmaps via gl.generateMipmap()

		return Math.log2( Math.max( image.width, image.height ) ) + 1;

	} else if ( texture.mipmaps.length > 0 ) {

		// user-defined mipmaps

		return texture.mipmaps.length;

	} else {

		// texture without mipmaps (only base level)

		return 1;

	}

}
```

Reply:
Indeed! 👍